mkdocs.yml

```diff
@@ -135,6 +133,9 @@ nav:
 
           - Releases: tre-developers/release.md
 
+      - Local Development: using-tre/local-development/index.md
```

Comment:
I think Local Development needs to go under the `Development` heading.

There is also a CI/CD link here:

![image](https://user-images.githubusercontent.com/17089773/191240544-3bc0ad88-1b52-476a-b8f9-eb5b12f6441e.png)

Makes sense to have a single CI/CD section under Technical Guide - CI/CD isn't just for development, but deployment.

Reply:
@marrobi - moved the "Local Development" under "Technical Guide" section and "Development" sub-section.

FO CI/CD - would you like me to create a new "sub-section" under "Technical Guide" which would include the whole sub-section of "Deploying the AzureTRE using CI/CD" and "CI / CD Pipelines" ?

Reply:
Are they not both the same thing?

Reply:
@marrobi, after having another read through CI/CD documentation my suggestion is to move "CI/CD Pipelines" sub-section under "Deploying the AzureTRE using CI/CD".
<img width="266" alt="image" src="https://user-images.githubusercontent.com/51514073/191681882-77629ab8-9b1e-4c25-aaf3-5a90f5d4a82b.png">

I think it is an appropriate place for this sub-section "Deploying the AzureTRE using CI/CD" does not have any content except for "Build Runners" sub-section.